### ISSUE TITLE
AudioLoader#loadAsync() cannot catch error as decoding audio wrong

### ISSUE THREAD

**Issue by DRL9** [mrdoob/three.js#21754]
**Describe the bug**

`AudioLoader#loadAsync()` cannot catch the error if loading a  invalid audio data. Instead, the console output a error message: **Uncaught (in promise) DOMException: Unable to decode audio data**

**To Reproduce**

1. Go to jsfiddle
2. Open Chrome devtool console
2. Click on button *click me* on jsfiddle
3. See error

***Code***

```js
const loader = new THREE.AudioLoader()
  loader.loadAsync('/').catch((e) => {
    console.log('loading wrong')
    console.error(e)
  })
```

***Live example***

* [jsfiddle-dev](https://jsfiddle.net/DRL9/epnz47dj/1/)

**Expected behavior**

Message **loading wrong** will show in the console.

**Platform:**

 - Device: [Desktop]
 - OS: [Linux]
 - Browser: [Chrome]
 - Three.js version: [dev, r127]

**Comment by DRL9** [comment]
Go to `src/loaders/AudioLoader.js`,
```js
context.decodeAudioData( bufferCopy, function ( audioBuffer ) {

    onLoad( audioBuffer );

} );
```
The code didn't add `errorCallback` or catch promise error. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/decodeAudioData)

**Comment by Mugen87** [comment]
This was already tried to fixed here: #20301
